lib/ui/painting.dart

```diff
@@ -3974,22 +3974,33 @@ class Vertices extends NativeFieldWrapperClass1 {
 
   /// Creates a set of vertex data for use with [Canvas.drawVertices], directly
   /// using the encoding methods of [new Vertices].
+  /// Note that this constructor uses a raw typed data list,
+  /// so it runs faster than Dart list version 
+  /// because it doesn't require any conversion from Dart list.
   ///
   /// The [mode] parameter must not be null.
   ///
   /// The [positions] list is interpreted as a list of repeated pairs of x,y
   /// coordinates. It must not be null.
+  /// The [positions] parameter is a list of triangular mesh vertices(xy).
```

Comment:
These 2 sentences should be combined, such as

The [positions] parameter is a list of triangular mesh vertices and is interpreted as ...

Reply:
(And no need to specify xy twice as the rest of the first sentence also mentions that they are x,y coordinates.)